metals/src/main/scala/scala/meta/internal/builds/GradleExperimentalBuildTool.scala

```diff
@@ -0,0 +1,75 @@
+package scala.meta.internal.builds
+
+import scala.meta.internal.metals.BuildInfo
+import scala.meta.internal.metals.Embedded
+import scala.meta.internal.metals.JavaBinary
+import scala.meta.internal.metals.UserConfiguration
+import scala.meta.io.AbsolutePath
+
+import coursierapi.Dependency
+import coursierapi.MavenRepository
+
+case class GradleExperimentalBuildTool(
+    userConfig: () => UserConfiguration,
+    projectRoot: AbsolutePath,
+) extends BuildTool
+    with BloopInstallProvider
+    with VersionRecommendation {
+
+  override def digest(workspace: AbsolutePath): Option[String] =
+    GradleDigest.current(projectRoot)
+
+  override def bloopInstallArgs(workspace: AbsolutePath): List[String] = {
+    val classpathSeparator = java.io.File.pathSeparator
+    val classpath = Embedded
+      .downloadDependency(
+        GradleExperimentalBuildTool.dependency,
+        repositories = Seq(GradleExperimentalBuildTool.gradleRepository),
+      )
+      .mkString(classpathSeparator)
+    List(
+      JavaBinary(userConfig().javaHome),
+      "-classpath",
+      classpath,
+      GradleExperimentalBuildTool.mainClass,
+    )
+  }
+
+  // current maximum supported version - just a placeholder, not tested for
+  override def version: String = "8.14.1"
+
+  override def minimumVersion: String = "2.12.0"
+
+  override def recommendedVersion: String = version
+
+  override def toString: String = "Gradle (Experimental)"
+
+  def executableName = GradleExperimentalBuildTool.name
```

Comment:
Currently it says "Multiple build definitions found. Which would you like to use", I would change it when it's essentially the same tool, so probably executableName could be the same for both versions and we can add an additional `def importName = executableName`.

Then if names are the same we could change the message to:
`Multiple build import methods are available. Which would you like to use? Read more about it in the [docs](buildTool.link)` and the link can point to https://scalameta.org/metals/docs/build-tools/gradle

What do you think about it?